
CondaError: Run 'conda init' before 'conda deactivate'

Traceback (most recent call last):
  File "/gpfs/radev/home/pl543/git-sources/checkpointing/examples/example2/ex2.py", line 38, in <module>
    mycpl = cpl.CPL()   
  File "/home/pl543/workspace/conda/torch_2.4/lib/python3.10/site-packages/cpl/cpl.py", line 52, in __init__
    self._email_user_handle_done = _config["email_types"]["user_handle_done"]
KeyError: 'user_handle_done'
srun: error: r4518u09n01: task 0: Exited with exit code 1
